packages/main/src/plugin/kubernetes/kubernetes-port-forward-service.ts

```diff
@@ -95,47 +90,28 @@ export class KubernetesPortForwardService implements IDisposable {
   }
 
   /**
-   * Each key is specific for a dedicated remotePort
+   * Return the identifier of the ForwardConfig
    * @param config
-   * @param mapping
    * @private
    */
-  private getPortForwardKey(config: ForwardConfig, mapping: PortMapping): string {
-    return `${config.namespace}/${config.name}/${config.kind}/${mapping.remotePort}`;
+  private getPortForwardKey(config: ForwardConfig): string {
+    return config.id;
   }
```

Comment:
question: do we need this private method anymore if we just need to use `.id` on the object ?

Reply:
I wanted to _reduce_ the number of change, but yes this method will disappear

Reply:
I will update the PR and remove the `getPortForwardKey`